lib/ansible/modules/cloud/vmware/vmware_vswitch.py

```diff
@@ -133,96 +140,260 @@ def __init__(self, module):
         self.nics = module.params['nics']
         self.mtu = module.params['mtu']
         self.state = module.params['state']
-        self.esxi_hostname = module.params['esxi_hostname']
+        esxi_hostname = module.params['esxi_hostname']
+
+        hosts = get_all_objs(self.content, [vim.HostSystem])
```

Comment:
Can we use the refactored logic of host/cluster here to simplify?

Reply:
Yes.